torch/distributed/rpc/backend_registry.py

```diff
@@ -14,8 +14,17 @@
     "BackendValue", ["construct_rpc_agent_options_handler", "init_backend_handler"]
 )
 
+
+def _backend_type_repr(self):
+    return "BackendType." + self.name
+
+
 # Create an enum type, `BackendType`, with empty members.
-BackendType = enum.Enum(value="BackendType", names={})
+BackendType = enum.Enum(
+    value="BackendType",
+    names={},
+    __repr__=_backend_type_repr
```

Comment:
How does this work.

It looks to me extra keyword arg will be dropped by this asterisk.

https://github.com/python/cpython/blob/293dd23477eef6e7c1b1e26b5bb2c1e0d79ac3c2/Lib/enum.py#L278

Reply:
Ah, yes you are right! I was doing `BackendType .__repr__ = _backend_type_repr` when generating the figure and then switched to this afterward. It was showing the same result, but I should have `make clean` the docs, after that I started to see errors. Let me update.